src/containers/my-quizzes/selectable-question-quiz-view/SelectableQuestionQuizView.tsx

```diff
@@ -28,13 +37,49 @@ const SelectableQuestionQuizView: FC<SelectableQuestionQuizViewProps> = ({
     )
   })
 
+  const onNext = () => {
+    if (!isLastQuestion) {
+      setSelectedIndex(selectedIndex + 1)
+    }
+  }
+
+  const onBack = () => {
+    if (!isFirstQuestion) {
+      setSelectedIndex(selectedIndex - 1)
+    }
+  }
+
   return (
     <Box>
       <Box sx={styles.selectableList}>{questionsNumberList}</Box>
       <SelectableQuestion
         question={questions[selectedIndex]}
         selectedIndex={selectedIndex}
       />
+      {!isSingleQuestion && (
+        <Box sx={styles.buttons}>
+          {!isFirstQuestion && (
+            <AppButton
+              onClick={onBack}
+              size={SizeEnum.ExtraLarge}
+              sx={{ display: 'flex', flexDirection: 'space-between' }}
```

Comment:
Maybe you should move this style to the style folder?

Reply:
This style was unnecessary, I have already removed it.